engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
If the model for our bottom-level `worker` actors is
create actor > send "start work" > actor works > actor sends "completion" back > actor destroyed
then create and start happen in the same place and it actually doesn't functionally make any difference where the params-sending happens.

However, if they're in the "start work" message, then the message to the router is exactly the same as the message to the worker. Which appeals to me sense of neatness.